OpenVBX/controllers/twiml.php

```diff
@@ -396,7 +400,35 @@ protected function dial_user_by_client_id($client_id, $options)
 		}
 		else
 		{
-			$this->reponse->say('Unknown client id: '.$user_id.'. Goodbye.');
+			$this->response->say('Unknown client id: '.$user_id.'. Goodbye.');
+			$this->response->hangup();
+		}
+	}
+	
+	/**
+	 * Join A Conference Call
+	 *
+	 * @param string $conference_name 
+	 * @param string $muted
+	 * @param string $beep
+	 * @return void
+	 */
+	protected function join_conference($conference_name,$muted,$beep)
+	{
+		if (strlen($conference_name)>2)
+		{
+			$confOptions = array(
+				'muted' => $muted,
+				'beep' => $beep,
+				'startConferenceOnEnter' => 'false'
+			);
+			
+			$dial = $this->response->dial(NULL, NULL);
```

Comment:
I'm changing the NULL to lower case but the reason I used uppercase was because I copied the function above it which also used upper case.  I'll just change all upper case NULL to lower in the file if that's what you prefer.